ngrinder-controller/src/main/java/org/ngrinder/starter/NGrinderControllerStarter.java

```diff
@@ -243,6 +245,23 @@ public static void main(String[] args) {
 		SpringApplication.run(NGrinderControllerStarter.class, args);
 	}
 
+	private static void checkTmpDirProperty() {
+		String userHome = System.getProperty("user.home");
+		String ngrinderHome = Config.getUserHome();
+		String javaTmpDir = System.getProperty("java.io.tmpdir");
+
+		if (!javaTmpDir.startsWith(userHome) && !javaTmpDir.startsWith(ngrinderHome)) {
+			System.out.print(CONSOLE_COLOR_RED);
+			System.out.println("####################################################################################################################");
+			System.out.println("# ERROR                                                                                                            #");
+			System.out.println("# Wrong runtime option. Please put tmpdir property like following.                                                 #");
```

Comment:
Please set java.io.tmpdir property like follwowing. tmpdir should be different from OS default tmpdir.